src/_actions/assetTypes.actions.js

```diff
@@ -40,3 +42,45 @@ export const createAssetTypeSuccess = assetType => (
 export const createAssetTypeFailure = error => (
   { type: CREATE_ASSET_TYPE_FAILURE, payload: error }
 );
+
+export const loadDropdownAssetTypes = () => (dispatch) => {
+  const pageNumber = 1;
+  axios
+    .get(`asset-types?page=${pageNumber}`)
+    .then((response) => {
+      const pages = Array.from(
```

Comment:
Once you find time, please look for me we look at this

Reply:
Noted. I will.

Reply:
@sachungo  As discussed, we will keep this implementation as we wait for the pagination correction in the backend.